src/rust/dbv2/migrations/2023-09-20-193413_add_limit_order_events/up.sql

```diff
@@ -43,16 +43,16 @@ CREATE TABLE
     PRIMARY KEY (txn_version, event_idx),
     time timestamptz NOT NULL,
     market_id NUMERIC(20) NOT NULL,
-    maker_address VARCHAR(70) NOT NULL,
-    maker_custodian_id NUMERIC(20) NOT NULL,
-    maker_order_id NUMERIC(40) NOT NULL,
-    maker_side BOOLEAN NOT NULL,
-    integrator_address VARCHAR(70) NOT NULL,
+    "user" VARCHAR(70) NOT NULL,
+    custodian_id NUMERIC(20) NOT NULL,
+    order_id NUMERIC(39) NOT NULL,
+    side BOOLEAN NOT NULL,
+    integrator VARCHAR(70) NOT NULL,
     initial_size NUMERIC(20) NOT NULL,
     price NUMERIC(20) NOT NULL,
-    restriction NUMERIC(20) NOT NULL,
-    self_match_behavior NUMERIC(20) NOT NULL,
-    posted_size NUMERIC(20) NOT NULL
+    restriction SMALLINT NOT NULL,
```

Comment:
You'll have to update the processor to accommodate the different type.

Reply:
Yes, that will come in a pr on the other repo.